docs/integrations/prefect-dbt/index.mdx

```diff
@@ -5,6 +5,9 @@ title: prefect-dbt
 With `prefect-dbt`, you can trigger and observe dbt Cloud jobs, execute dbt Core CLI commands, and incorporate other tools, such as [Snowflake](integrations/prefect-snowflake/index), into your dbt runs.
 Prefect provides a global view of the state of your workflows and allows you to take action based on state changes.
 
+Prefect integrations may provide pre-built [blocks](/latest/develop/blocks/), [flows](/latest/develop/flows/), or [tasks](/latest/develop/tasks/) for interacting with external systems.
+Block types in this library allow you to do such things as run a dbt Cloud job or execute a dbt Core command.
```

Comment:
```suggestion
Block types in this library allow you to do things such as run a dbt Cloud job or execute a dbt Core command.
```

Reply:
Looks like either option works: https://ell.stackexchange.com/questions/14544/difference-between-such-things-as-and-things-such-as